pallets/xcmp-queue/src/lib.rs

```diff
@@ -167,6 +167,86 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Overwrites the number of pages of messages which must be in the queue for the other side to be told to
+		/// suspend their sending.
+		///
+		/// - `origin`: Must pass `Root`.
+		/// - `new`: Desired value for `QueueConfigData.suspend_value`
+		#[pallet::weight(10_000_000 as Weight + T::DbWeight::get().reads_writes(1, 1))]
```

Comment:
From where is this `10_000_000` coming?

Reply:
Rough estimation based on Polkadot benchmarks [here](https://github.com/paritytech/polkadot/blob/b3d0778e9e18bf3097223a3af2c8e90d0351c217/runtime/polkadot/src/weights/runtime_parachains_configuration.rs#L50)
Since Keith mentioned [here](https://github.com/paritytech/cumulus/issues/801#issuecomment-1019542099) that he doesn't think we need to write a benchmark, I figured it would be a good default value. https://github.com/paritytech/cumulus/pull/931#discussion_r791544504

Reply:
Isn't it straight forward to write a benchmark for this? 

CC @shawntabrizi

Reply:
It is, I'm not suggesting to _not_ write anything, but if it's too daunting of a task to do so, then putting a `DbWeight::get().read_write` there is a good compromise.

Reply:
> `DbWeight::get().read_write`

I mean the `10_000_000`.

Reply:
@bkchr `10_000_000` not needed? Just `DbWeight::get( ).read_write` is sufficient?

Reply:
Let's keep them, but I will create an issue to create proper benchmarking.

Reply:
https://github.com/paritytech/cumulus/issues/961 created it